internal/goldendataset/pict_metrics_gen.go

```diff
@@ -74,11 +74,11 @@ func pictToCfg(inputs PICTMetricInputs) MetricsCfg {
 		cfg.IsMonotonicSum = false
 	case MetricTypeDoubleGauge:
 		cfg.MetricDescriptorType = pdata.MetricDataTypeDoubleGauge
-	case MetricTypeMonotonicDoubleSum:
-		cfg.MetricDescriptorType = pdata.MetricDataTypeDoubleSum
+	case MetricTypeMonotonicSum:
```

Comment:
Ok my point was that I don't think we should do this change since, the Sum will have 2 possible values (Int or Double). So for this is good to keep Double in the name to signal that Sum will have Double values in the NumberDataPoint.

Reply:
Ah I see. Thanks for clarifying!

Reply:
pict changes reverted.